recipes/jb-everything/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "jb-everything" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jb-everything-{{ version }}.tar.gz
+  sha256: efe53c547aae8794372817182db86822e039349ad6099ce4c9f0cd2e37f1c21a
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
Is there a reason to skip windows and not just use `noarch: python`?

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

I'm not sure if every dependency is available on windows, but if they are, we should strive to make thie `noarch`.

Reply:
Not every dependency is available on Windows - I've added a comment to that effect.